Libraries/Animated/src/NativeAnimatedHelper.js

```diff
@@ -159,7 +160,7 @@ const API = {
  * In general native animated implementation should support any numeric property that doesn't need
  * to be updated through the shadow view hierarchy (all non-layout properties).
  */
-const STYLES_WHITELIST = {
+const SHADOW_VIEW_PROPS = {
```

Comment:
I think this name is wrong, it is the other way around. These props do not require to be update through the shadow view. Maybe call it something like `NON_LAYOUT_PROPS` or `UI_THREAD_PROPS`.

Reply:
Moved to native side. Added deprecation warnings when trying to add extra props to these lists.